etl/steps/data/garden/hmd/2023-09-19/hmd.meta.yml

```diff
@@ -59,7 +59,7 @@ tables:
       number_person_years_remaining:
         title: Number of person-years remaining
         unit: person-years
-        description_short: Number of person-years remaining after exact age x.
+        description_short: Number of person-years remaining after a given age.
```

Comment:
There is a commented `description_short` below. Is that intentional?

Reply:
can remove